test/lib/modules/apostrophe-salesforce-connect/salesforce-metadata.xml

```diff
@@ -0,0 +1,17 @@
+<?xml version="1.0" encoding="UTF-8"?>
```

Comment:
See above, move all tests to a private test project. OR, use env vars to determine what cert file folder to use, and find the files there. Then you don't need a separate project but people need to get their own certs if they really want to test this, which makes sense because they would also need their own SF world to test against.